tools/lint-all

```diff
@@ -13,6 +13,8 @@ from collections import defaultdict
 from six.moves import filter
 from six.moves import map
 
+import lister
```

Comment:
maybe makes sense for this to be a `from . import lister`?  It's a little safer in the presence of potential namespace collisions with something installed in the standard library.